frontend/src/api/operatePreferenceData.test.ts

```diff
@@ -26,4 +26,18 @@ describe ('use of updateNickname method', () => {
     expect(operateUserDataRequest.updateNickname).toHaveBeenCalled()
     spy.mockRestore()
   })
+
+
+  describe ('test use of post user customization', () => {
+    it('test usage of postcustomization method of CustomizeRequest class', () => {
+      const changeUserNotificationSpy = jest.spyOn(operateUserDataRequest, 'changeUserNotification')
```

Comment:
Unfortunately it's useless test: you create a spy and call it directly. It has no sense. Which case do you want to cover with this test?

Reply:
I may not undesrtand you correctly. This is a test from jest documentation, and it test that a method was called. I added mockImplementation to avoid info been sent further and returns from other components been seen.